python-package/lightgbm/basic.py

```diff
@@ -2263,12 +2260,18 @@ def feature_importance(self, importance_type='split', iteration=-1):
             How the importance is calculated.
             If "split", result contains numbers of times the feature is used in a model.
             If "gain", result contains total gains of splits which use the feature.
+        iteration : int or None, optional (default=None)
```

Comment:
This argument was not documented at all. I've changed its default value for the consistency with other similar params in other methods.